docs/Installation.md

```diff
@@ -183,15 +193,17 @@ Install, configure and start database engine:
   sudo systemctl start postgresql
   ```
 
-Initialize Zonemaster database (unless you keep an old database):
+Initialize Zonemaster database and user (unless you keep an old database):
 
 ```sh
-sudo -u postgres psql -f $(perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")')/initial-postgres.sql
+sudo -u postgres createuser -P zonemaster
 ```
 
-> **Note:** This creates a database called `zonemaster`, as well as a user called
-> "zonemaster" with the password "zonemaster" (as stated in the config file).
-> This user has just enough permissions to run the backend software.
+This will ask for a password that should be copied in `backend_config.ini`.

```

Comment:
Why is that needed now when it was not needed before?

Reply:
not related to this PR